tidypolars/funs.py

```diff
@@ -131,6 +131,22 @@ def as_string(x):
     x = _col_expr(x)
     return x.cast(pl.Utf8)
 
+def as_factor(x, levels):
+    """
+    Convert to factor. Defaults to Utf8.
```

Comment:
Update to `Convert to categorical.` and remove `Defaults to Utf8.`

Reply:
Agree